packages/astro/src/vite-plugin-astro-postprocess/index.ts

```diff
@@ -46,13 +46,7 @@ export default function astro(_opts: AstroPluginOptions): Plugin {
 						const firstArgStart = node.arguments[0].start;
 						const firstArgEnd = node.arguments[0].end;
 						const lastArgEnd = node.arguments[node.arguments.length - 1].end;
-						let firstArg = code.slice(firstArgStart, firstArgEnd);
-						// If argument is template literal, try convert to a normal string.
-						// This is needed for compat with previous recast strategy.
-						// TODO: Remove in Astro 2.0
-						if (firstArg.startsWith('`') && firstArg.endsWith('`') && !firstArg.includes('${')) {
-							firstArg = JSON.stringify(firstArg.slice(1, -1));
-						}
+						const firstArg = code.slice(firstArgStart, firstArgEnd);
```

Comment:
The intent here was to remove support for:

```js
Astro.glob(`./foo/*.md`) // using template literals
```

which transforms to `import.meta.glob`, but it looks like Vite 4 now supports template literal too so this is no longer a breaking change to note in the changeset.